### PR TITLE
Keyboard_node merge after tf_lidar

### PR DESCRIPTION
**Pull request by roym899**
This adds a keyboard node which allows to control the robot via WASD. Q is used to quit the node.

This has been tested and works on the robot.

### CODE REVIEW

File: er_wheels/src/er_motor_controller_node.cpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const int controller_frequency = 25;
+
+const double ticks_per_rev = 3591.84/4;
+
+double omega = 0;
+
+double omega_goal = 0;
+
+// use PI controller, since 25hz is probabaly to low to use d controller in a meaningful way
+double p = 0;
```
Comment (meeshen) [inline er_wheels/src/er_motor_controller_node.cpp:18]: #edited: this becomes an I-controller only

File: er_wheels/src/er_wheel_odometry_node.cpp [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  angular_velocity_[MotorSide::Left] = 0;
+  angular_velocity_[MotorSide::Right] = 0;
+  // note messages are initialized with 0, so intiially the /base_frame will align with the /map frame
+}
+
+WheelOdometryNode::~WheelOdometryNode() {
+
+}
+
+void WheelOdometryNode::run_node() {
```
Comment (meeshen) [inline er_wheels/src/er_wheel_odometry_node.cpp]: as mentioned earlier, could be good to break it up a little

File: er_wheels/src/er_motor_controller_node.cpp (hunk 1 above)
Comment (roym899) [reply er_wheels/src/er_motor_controller_node.cpp:18]: Right now I only works pretty well when driving around.
We can tune that later though or we do a feed forward controller if we have time. We can discuss that when we meet.

File: er_wheels/src/er_wheel_odometry_node.cpp (hunk 2 above)
Comment (roym899) [reply er_wheels/src/er_wheel_odometry_node.cpp]: Addressed in the latest commit.

**Comment by roym899** [comment]
I addressed your comment and also fixed a bug related to omega_avg == 0, which caused the odometry to jump. It seems to work now.

**Pull request merged**